display-brightness-ddcutil@themightydeity.github.com/metadata.json

```diff
@@ -7,7 +7,6 @@
     "gettext-domain": "display-brightness-ddcutil",
     "version": 25,
     "shell-version": [
-        "40",
-        "41"
```

Comment:
You cannot remove the old one, as it works in 40 and 41 as well. You can just add 42

Reply:
Is there any other reason than to make it look nice?

Reply:
Libadwaita was originally planned for version 41 but was delayed to 42, so I don't expect these changes to work in 40 and 41.

We can have two `prefs.ui` and check for Shell's version as extensions supporting versions 3.3X do for GTK3 and GTK4 ui files. Another way would be to have two branches and publish updates with different version numbers.
In any case, we would have to adjust both `prefs.ui` to add any new features.